webui/react/src/pages/TrialDetails/F_TrialDetailsOverview.tsx

```diff
@@ -64,17 +115,43 @@ const TrialDetailsOverview: React.FC<Props> = ({ experiment, trial }: Props) =>
       }
 
       out.push({
+        onPointClick,
+        plugins: [
+          closestPointPlugin({
+            checkpointsDict,
+            diamond: true,
+            onPointClick,
+            yScale: 'y',
+          }),
+          drawPointsPlugin(checkpointsDict),
+          tooltipsPlugin({
+            getXTooltipHeader(xIndex) {
+              const xVal = dataForAxis?.[xIndex]?.[0];
+              if (xVal === undefined) return '';
+              const checkpoint = checkpointsDict?.[Math.floor(xVal)];
+              if (!checkpoint) return '';
+              return '<div class="">⬦ Best Checkpoint <em>(click to view details)</em> </div>';
+            },
+            isShownEmptyVal: false,
+            seriesColors: series.map((s) => s.color ?? '#009BDE'),
+          }),
+        ],
         series,
         title: trainingMetric.name.replace(TRAIN_PREFIX, '').replace(VAL_PREFIX, ''),
+        xAxis,
+        xLabel: 'Batch',
```

Comment:
Previously we've been using Batch**es** as the x-axis 🤔

Reply:
I can revert

Reply:
I don't know if we have latest figma showing one way or the other